website/content/en/docs/kubebuilder/advanced-topics.md

```diff
@@ -60,14 +61,15 @@ import (
   routev1 "github.com/openshift/api/route/v1"
```

Comment:
@theishshah This doc is still referencing the old SDK layout filepath. Can't comment there but a few lines above it says:
```
Call the `AddToScheme()` function for your 3rd party resource and pass it the Manager's scheme via `mgr.GetScheme()`
in `cmd/manager/main.go`.
```
Where `cmd/manager/main.go` is now `main.go` in kubebuilder's layout.

Can you please proof read the doc again for other such references.